package.json

```diff
@@ -385,5 +385,6 @@
   },
   "nx": {
     "includedScripts": []
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
no idea where this came from but please revert this change

Reply:
Done @Hotell , I have no idea where that came from 👀

Reply:
> no idea where this came from but please revert this change

Just found out where that came from: https://github.com/nodejs/corepack/issues/485 , seems like corepack is automatically adding `packageManager` everytime `yarn` command runs, `COREPACK_ENABLE_AUTO_PIN="0"` environment variable is required in this case